libraries/rush-lib/package.json

```diff
@@ -5,7 +5,7 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/microsoft/rushstack.git",
-    "directory": "apps/rush-lib"
+    "directory": "libraries/rush-lib"
```

Comment:
We'll need a corresponding fix in https://github.com/microsoft/rushstack-websites

Reply:
https://github.com/microsoft/rushstack-websites/pull/45